yaml2ics.py

```diff
@@ -45,18 +45,20 @@ def utcnow():
 # This function can be used to add a list of e.g. exception dates (EXDATE) or
 # recurrence dates (RDATE) to a reoccurring event
 def add_recurrence_property(
-    event: ics.Event, property_name, dates: map, tz: datetime.tzinfo = None
+    event: ics.Event, property_name, dates: map, tz: datetime.tzinfo = dateutil.tz.UTC
 ):
     event.extra.append(
         ics.ContentLine(
             name=property_name,
-            params={"TZID": [str(ics.Timezone.from_tzinfo(tz))]} if tz else None,
+            params={"TZID": [str(ics.Timezone.from_tzinfo(tz))]},
             value=",".join(dates),
         )
     )
 
 
-def event_from_yaml(event_yaml: dict, tz: datetime.tzinfo = None) -> ics.Event:
+def event_from_yaml(
+    event_yaml: dict, tz: datetime.tzinfo = dateutil.tz.UTC
```

Comment:
I think either change the default, or change the handling below, but not both.

Reply:
OK, after #125 this becomes the crux of the current PR. We should document this in the README.